components/zoho_campaigns/actions/add-contact-to-mailing-list/add-contact-to-mailing-list.mjs

```diff
@@ -0,0 +1,93 @@
+import app from "../../zoho_campaigns.app.mjs";
+
+export default {
+  type: "action",
+  key: "zoho_campaigns-add-contact-to-mailing-list",
+  name: "Add Contact to Mailing List",
+  version: "0.0.1",
+  description: "You can use this API to add contacts to your mailing lists. [See the documentation](https://www.zoho.com/campaigns/help/developers/contact-subscribe.html)",
+  props: {
+    app,
+    listkey: {
+      propDefinition: [
+        app,
+        "mailingList",
+      ],
+      label: "Mailing List",
+      description: "List Key to send a subscription mail to contacts of the list.",
+    },
+    contactEmail: {
+      type: "string",
+      label: "Contact Email",
+      description: "Contact email can be added.",
+    },
+    contactFirstName: {
+      type: "string",
+      label: "Contact First Name",
+      description: "Contact first name can be added.",
+      optional: true,
+    },
+    contactLastName: {
+      type: "string",
+      label: "Contact Last Name",
+      description: "Contact last name can be added.",
+      optional: true,
+    },
+    source: {
+      type: "string",
+      label: "Source",
+      description: "Contact source can be added.",
+      optional: true,
+    },
+    topic_id: {
```

Comment:
use cammelCase

Reply:
I am using snake case to be able to use destructor in `run`. I would need to write more code if I use cammelCase for everything